docs/operational_guide/namespace_configuration.md

```diff
@@ -128,6 +128,16 @@ This is the most important value to consider when tuning the performance of an M
 
 Can be modified without creating a new namespace: `no`
 
+Below are recommendations for block size based on retention:
+
+| Retention | Block Size |
+|-----------|------------|
+| 12h       | 30m        |
+| 24h       | 1h         |
+| 168h      | 2h         |
+| 720h      | 12h        |
+| 8760h     | 24h        |
```

Comment:
Hmm, the larger ones don't seem right? I know we wanted to match these with the database create call, but 24 hours seem small for 1 year and we also don't take into account the resolution at all here. Can we make a TODO to update the create to take scrape interval or aggregation resolution into account?

Also, thoughts on index block size vs TS block size? @robskillington I've seen some folks with good success using larger index block sizes than TS block sizes, esp is there's not a huge amount of new IDs. Maybe we can configure those into create and update them here too.

Reply:
Yeah, I think we want to be consistent with the database create call which uses this for resolution:
```
	idealDatapointsPerBlock           = 720
	blockSizeFromExpectedSeriesScalar = idealDatapointsPerBlock * int64(time.Hour)

	value := r.BlockSize.ExpectedSeriesDatapointsPerHour
	blockSize = time.Duration(blockSizeFromExpectedSeriesScalar / value)
```
But to be honest, if we push users towards the database create api, no one should really need to know about this.